lib/structuraid_core/engineering/analysis/footing/utils/data.rb

```diff
@@ -0,0 +1,36 @@
+module StructuraidCore
+  module Engineering
+    module Analysis
+      module Footing
+        module Utils
+          module Data
```

Comment:
what is the purpose of this module? I don't think `Data` is the best name for this

Reply:
I don´t like it too, I changed it to `BasicGeometry`, and If we use it, we can implement this util into `CentricIsolated`too

Reply:
Let's make sure to create a ticket for this refactor on CentricIsolated